internal/core/src/query/visitors/ExecExprVisitor.cpp

```diff
@@ -186,7 +188,10 @@ AppendOneChunk(BitsetType& result, const FixedVector<bool>& chunk_res) {
     // Append a value once instead of BITSET_BLOCK_BIT_SIZE times.
     auto AppendBlock = [&result](const bool* ptr, int n) {
         for (int i = 0; i < n; ++i) {
-            BitSetBlockType val = 0;
+#if defined(USE_DYNAMIC_SIMD)
+            auto val = milvus::simd::get_bitset_block(ptr);
+#else
+            BitsetBlockType val = 0;
```

Comment:
`val` will be re-defined if `USE_DYNAMIC_SIMD` is true.

Reply:
Sorry, ignore this. I don't notice that line 194 is controlled by the `else` block.